### PR TITLE
fix: don't use update-notifier

### LINKED ISSUE #2752: Stop using update-notifier
We have all the necessary code for making requests to the registry. We don't need update-notifier. We just need a place where we'll save some state between executions of pnpm. So that we don't check for new versions on each request. (we'll probably use that state storage for implementing #2694 as well)

### PR DESCRIPTION
**Pull request by zkochan**
close #2752

### CODE REVIEW

**Review by statianzo — APPROVED** [review]

File: packages/pnpm/package.json [hunk 1]
```diff
@@ -16,14 +16,13 @@
     "bin"
   ],
   "optionalDependencies": {
-    "graceful-fs": "4.2.4",
```
Comment (ExE-Boss) [inline packages/pnpm/package.json:19]: And `graceful‑fs` is being removed from `optionalDependencies` because ???

**Comment by AHM435** [comment]
So what


________________________________
Von: ExE Boss ***@***.***>
Gesendet: Tuesday, March 23, 2021 6:29:46 AM
An: pnpm/pnpm ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: Re: [pnpm/pnpm] fix: don't use update-notifier (#3270)


@ExE-Boss commented on this pull request.

________________________________

In packages/pnpm/package.json<https://github.com/pnpm/pnpm/pull/3270#discussion_r599280208>:

> @@ -16,14 +16,13 @@

     "bin"

   ],

   "optionalDependencies": {

-    "graceful-fs": "4.2.4",



And graceful‑fs is being removed from optionalDependencies because ???

File: packages/pnpm/package.json (hunk 1 above)
Comment (zkochan) [reply packages/pnpm/package.json:19]: graceful-fs was removed from all dependencies in the repository (as one of the changes in pnpm v6). Not sure why it was still in pnpm CLI.

**Comment by nikoladev** [comment]
FYI, `@types/update-notifier` was not removed: [source][pkg].

Was this on purpose or an oversight?

[pkg]: https://github.com/pnpm/pnpm/blob/1a9b4f81233cbfbebb667d71ad4ae15c951aa3cd/packages/pnpm/package.json#L67

**Comment by zkochan** [comment]
Thanks, it was an oversight.

**Pull request merged**